commit cca86656508d12f998e11ee7f71b8a40864af3b4
Author: Benjamin Tissoires <benjamin.tissoires@redhat.com>
Date:   Mon Mar 7 11:02:38 2016 +0100

    HID: multitouch: force retrieving of Win8 signature blob
    
    commit 45c5c6828214605eaefa6755c47bd1a2c7eb203e upstream.
    
    The Synaptics 0x11e5 over I2C found in the Asus T100-CHI requires to
    fetch the signature blob to actually start sending events.
    
    With this patch, we should be close enough to the Windows driver which
    checks the content of the blob at plugin to validate or not the
    touchscreen.
    
    Link: https://bugzilla.kernel.org/show_bug.cgi?id=113481
    Fixes: 6d4f5440 ("HID: multitouch: Fetch feature reports on demand for Win8 devices")
    Signed-off-by: Benjamin Tissoires <benjamin.tissoires@redhat.com>
    Signed-off-by: Jiri Kosina <jkosina@suse.cz>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/drivers/hid/hid-multitouch.c b/drivers/hid/hid-multitouch.c
index 2b8ff18..c5ec4f9 100644
--- a/drivers/hid/hid-multitouch.c
+++ b/drivers/hid/hid-multitouch.c
@@ -396,6 +396,11 @@ static void mt_feature_mapping(struct hid_device *hdev,
 			td->is_buttonpad = true;
 
 		break;
+	case 0xff0000c5:
+		/* Retrieve the Win8 blob once to enable some devices */
+		if (usage->usage_index == 0)
+			mt_get_feature(hdev, field->report);
+		break;
 	}
 }
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1328_cca86656508d12f998e11ee7f71b8a40864af3b4.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1328_cca86656508d12f998e11ee7f71b8a40864af3b4.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
